### PR TITLE
added courses on free-course-en

### PR DESCRIPTION
**Pull request by m1racul**
Added courses of AutoIt, Bootstrap, C sharp, Dart, git, Node.js on free-course-en

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -148,6 +148,7 @@
 ### AutoIt
 
 * [AutoIt Scripting Tutorial](https://www.youtube.com/playlist?list=PL4Jcq5zn02jKpjX0nqI1_fS7mEEb5tw6z) - TutsTeach (YouTube)
+* [AutoIt Turtorial](https://www.youtube.com/watch?v=82ifMPijghw&list=PL1DCD109B801D0DE6) - 403forbidden403 (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:151]: We should fix this acording to CONTRIBUTE guidelines
- Use playlist url
- Fix wrong title

Maybe this hunk solve all:
```markdown
* [AutoIt Tutorials](https://www.youtube.com/playlist?list=PL1DCD109B801D0DE6) - 403forbidden403 (YouTube)
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -168,6 +169,7 @@
 * [Learn Bootstrap 4 for free](https://scrimba.com/learn/bootstrap4) - Neil Rowe (Scrimba)
 * [Learn Bootstrap 4 in this free 10-part course](https://www.freecodecamp.org/news/want-to-learn-bootstrap-4-heres-our-free-10-part-course-happy-easter-35c004dc45a4/) - Per Harald Borgen (Freecodecamp)
 * [Rapid website design with Bootstrap](https://www.udemy.com/course/responsive-website-design) - Laurence Svekis (Udemy)
+* [Learn Bootstrap 5](https://www.youtube.com/watch?v=iJKCj8uAHz8) - freeCodeCamp (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:172]: :information_source: Not a playlist but 5h video with timeline in player bar

:warning: Please use original title and try to give also credits to instructor

```markdown
* [Learn Bootstrap 5 and SASS by Building a Portfolio Website - Full Course](https://www.youtube.com/watch?v=iJKCj8uAHz8) - Patrick Muriungi, freeCodeCamp (YouTube)
```

And of course, ensure that listing alphabetization is preserved

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -183,6 +185,8 @@
 * [C# 101](https://channel9.msdn.com/Series/CSharp-101) - Scott Hanselman, Kendra Havens Microsoft.
 * [C# Tutorial - Full Course for Beginners](https://www.youtube.com/watch?v=GhQdlIFylQ8) - Mike Dane - (freeCodeCamp)
 * [Learn how to program: C#](https://www.learnhowtoprogram.com/c) - Epicodus Inc.
+* [C# Tutorial For Beginners](https://www.youtube.com/watch?v=gfkTfcpWqAY) - Programming with Mosh (Youtube)
+* [C# Programming All-in-One Tutorial Series](https://www.youtube.com/watch?v=qOruiBrXlAw) - Caleb Curry (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:189]: :information_source: Not a playlists but with timeline, one 1h (the min) other 6h

:warning: Please satisfy CONTRIBUTE guidelines:
```markdown
* [C# Tutorial For Beginners - Learn C# Basics in 1 Hour](https://www.youtube.com/watch?v=gfkTfcpWqAY) - Moshfegh Hamedani, Programming with Mosh (YouTube)
* [C# Programming All-in-One Tutorial Series (6 HOURS!)](https://www.youtube.com/watch?v=qOruiBrXlAw) - Caleb Curry (YouTube)
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -216,6 +220,7 @@
 * [Dart Course for Beginners](https://www.udemy.com/course/dartlang) (Udemy)
 * [Dart Programming Tutorial - Full Course](https://www.youtube.com/watch?v=Ej_Pcr4uC2Q) - Mahmud Ahsan (FreeCodeCamp)
 * [Dart Programming Tutorial \| Learn the Dart for Flutter](https://www.udemy.com/course/dart-programming-tutorial-learn-the-dart-for-flutter) (Udemy)
+* [Dart Programming in 4 hours | Full beginners tutorial](https://www.youtube.com/watch?v=5xlVP04905w) - Mike Dane (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:223]: :information_source: Not a playlist but 5h video with timeline in player bar

Minor markdown faults
```markdown
* [Dart Programming in 4 hours \| Full beginners tutorial](https://www.youtube.com/watch?v=5xlVP04905w) - Mike Dane (YouTube)
```

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -325,6 +330,7 @@
 * [GitHowTo](https://githowto.com)
 * [How to Use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775) (Udacity)
 * [Introduction to Git and GitHub](https://www.coursera.org/learn/introduction-git-github) - Google (Coursera)
+* [Git Tutorial for Beginners](https://www.youtube.com/watch?v=8JJ101D3knE&t=626s) - Programming with Mosh (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:333]: :information_source: Not a playlist but 1h10m video with timeline in player bar

:warning: Please, time cursor is not acceptable. Complete creator attributions and resource title too.

```markdown
* [Git Tutorial for Beginners: Learn Git in 1 Hour](https://www.youtube.com/watch?v=8JJ101D3knE) - Moshfegh Hamedani, Programming with Mosh (YouTube)
```

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -463,6 +469,7 @@
 * [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) - John Smilga, Coding Addict (YouTube)
 * [Node.js API Development for Beginners](https://www.udemy.com/course/node-js-api-tutorial/) (Udemy)
 * [Node.js Basics](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa-d0iZg41U2RDqECol9C5B) - Java Brains (Youtube)
+* [Node.js Tutorial for Beginners](https://www.youtube.com/watch?v=TlB_eWDSMt4) - Programming with Mosh (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:472]: :information_source: Not a playlist but 1h20m video with timeline in player bar

:warning: Some issues with creator attribution completion and right resource title

```markdown
* [Node.js Tutorial for Beginners: Learn Node in 1 Hour](https://www.youtube.com/watch?v=TlB_eWDSMt4) - Moshfegh Hamedani, Programming with Mosh (YouTube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Remember also satisfy linter lists alphabetization rule:

```logs
Run fpb-lint ./courses/
/courses/free-courses-en.md
   164:1-172:92  warning  Alphabetical ordering: swap l.172 and l.171  alphabetize-lists  remark-lint
  185:1-189:115  warning  Alphabetical ordering: swap l.188 and l.187  alphabetize-lists  remark-lint
  185:1-189:115  warning  Alphabetical ordering: swap l.189 and l.188  alphabetize-lists  remark-lint
  220:1-223:125  warning  Alphabetical ordering: swap l.223 and l.222  alphabetize-lists  remark-lint
  324:1-333:117  warning  Alphabetical ordering: swap l.333 and l.332  alphabetize-lists  remark-lint

⚠ 5 warnings
Error: Process completed with exit code 1.
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (m1racul) [reply courses/free-courses-en.md:151]: thanks

File: courses/free-courses-en.md (hunk 2 above)
Comment (m1racul) [reply courses/free-courses-en.md:171]: thanks

File: courses/free-courses-en.md (hunk 3 above)
Comment (m1racul) [reply courses/free-courses-en.md]: thanks

File: courses/free-courses-en.md (hunk 4 above)
Comment (m1racul) [reply courses/free-courses-en.md]: thanks

File: courses/free-courses-en.md (hunk 5 above)
Comment (m1racul) [reply courses/free-courses-en.md]: thanks

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -181,8 +183,10 @@
 ### C Sharp
 
 * [C# 101](https://channel9.msdn.com/Series/CSharp-101) - Scott Hanselman, Kendra Havens Microsoft.
+* [C# Programming All-in-One Tutorial Series](https://www.youtube.com/watch?v=qOruiBrXlAw) - Caleb Curry (Youtube)
 * [C# Tutorial - Full Course for Beginners](https://www.youtube.com/watch?v=GhQdlIFylQ8) - Mike Dane - (freeCodeCamp)
 * [Learn how to program: C#](https://www.learnhowtoprogram.com/c) - Epicodus Inc.
+* [C# Tutorial For Beginners](https://www.youtube.com/watch?v=gfkTfcpWqAY) - Programming with Mosh (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:189]: Maybe Mr. "C" would like to go up stairs :smile:

File: courses/free-courses-en.md [hunk 8]
```diff
@@ -323,6 +328,7 @@
 * [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
 * [GIT for beginners](https://www.eduonix.com/git-for-beginners) - Maksym Rudnyi (Eduonix Learning Solutions) *(account or email address required)*
 * [GitHowTo](https://githowto.com)
+* [Git Tutorial for Beginners](https://www.youtube.com/watch?v=8JJ101D3knE&t=626s) - Programming with Mosh (Youtube)
```
Comment (davorpa) [inline courses/free-courses-en.md:331]: :information_source: It is a good opportunity for these GIT colleagues to exchange their seats.

**Review by davorpa — CHANGES_REQUESTED** [review]
:warning: Solving linter warnings...

```logs
Run fpb-lint ./courses/
  185:1-189:109  warning  Alphabetical ordering: swap l.189 and l.188  alphabetize-lists  remark-lint
/home/runner/work/free-programming-books/free-programming-books/courses/free-courses-en.md
  324:1-333:111  warning  Alphabetical ordering: swap l.331 and l.330  alphabetize-lists  remark-lint

⚠ 2 warnings
Error: Process completed with exit code 1.
```

Next time, it would be nice if you knew how to check it out by yourself :wink:. See [HOWTO](https://github.com/EbookFoundation/free-programming-books/blob/master/HOWTO.md) guidelines.

**Pull request merged**